extensions/todo-list/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Todo List Changelog
 
+## Fix Handling URLs Without a Scheme (2022-03-17)
+
+Previously, URLs without a scheme would crash the extension -- these are now handled by adding the `http` scheme if there is no scheme.
```

Comment:
```suggestion
Previously, URLs without a scheme would crash the extension -- these are now handled by adding the `https` scheme if there is no scheme.
```

Reply:
Shouldn't it be secure?

Reply:
I was considering that some urls might not have https, but urls that are secure will usually redirect from http to https, but I can change it